lib/helpers.js

```diff
@@ -60,11 +60,13 @@ module.exports.getFirstWord = function (message) {
 };
 
 module.exports.isYesResponse = function (message) {
-  const trimmed = this.getFirstWord(message.toLowerCase().trim());
+  const trimmedMsg = message.toLowerCase().trim();
   const yesResponses = process.env.GAMBIT_YES_RESPONSES ?
     process.env.GAMBIT_YES_RESPONSES.toLowerCase().split(',') : ['yes'];
 
-  return yesResponses.indexOf(trimmed) >= 0;
+  const matchRegex = new RegExp(`^(${yesResponses.join('|')})$`);
```

Comment:
Minor, but it might be nice to leave a comment to translate this RegExp into readable. Will this find a yes response anywhere in the string? e.g. "I'm thinking about it and the answer is YES"